zhaquirks/lixee/zlinky.py

```diff
@@ -41,7 +55,7 @@ class ZLinkyTIC(CustomDevice):
                     Metering.cluster_id,
                     MeterIdentification.cluster_id,
                     ElectricalMeasurement.cluster_id,
-                    0xFF66,  # Manufacturer Specific
+                    ZLINKY_MANUFACTURER_CLUSTER_ID,
```

Comment:
Since `ZLinkyTICManufacturerCluster` has a `cluster_id` attribute, could this line be changed to `ZLinkyTICManufacturerCluster.cluster_id` just to keep the notation consistent ?

Reply:
Done.